engines/agi/detection.cpp

```diff
@@ -285,6 +286,7 @@ ADDetectedGame AgiMetaEngineDetection::fallbackDetect(const FileMap &allFilesXXX
 		// is alive (as else the string storage is modified/deleted).
 		g_fallbackDesc.desc.gameId = _gameid.c_str();
 		g_fallbackDesc.desc.extra = _extra.c_str();
+		g_fallbackDesc.desc.guiOptions = GAMEOPTIONS_FANMADE_MOUSE_VGA;
```

Comment:
This is wrong. AGI games were made for EGA in PC versions. There exist about 2-3 fan made games with a hacked interpreter, which show VGA graphics. We can't allow this VGA graphics option for all unknown AGI games, just because 3 known fan made games with hacked interpreters use this feature. Please remove this.

Reply:
I was assuming that if the game is non-recognized, then it might as well be anything, really - so the option set should be possibly broad. Such a VGA-hacked fanmade game, when tinkered with, may fail detection checks and become "-fallback" - and it'll lose its VGA option. Would that be expected and correct?

Reply:
Games matched through the fallback detector are assigned to game options that apply to most games. The VGA setting only applies to a very small subset of fan made games, so it doesn't make sense to enable this setting for any unknown game.

Reply:
Or perhaps there's a different setting I missed, that would do the above - enable ALL possible options for maximum flexibility, you-clearly-know-what-you're-doing-so-have-fun?